src/controller.py

```diff
@@ -29,10 +28,21 @@
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
+error_codes = {
+    32601: "method not found",
+    32602: "invalid params",
+    32603: "nonce not found",
+    32604: "cred def id not found",
+    32605: "unsupported platform",
+    32606: "invalid challenge",
+    32607: "unable to cache nonce",
+}
+
 
 def handle_drpc_request(drpc_request, connection_id):
     handler = {
         "request_nonce": handle_drpc_request_nonce,
+        "request_attestation": handle_drpc_request_challange,
```

Comment:
Not sure if this is a mispelling of challenge or if challange is a different word

Reply:
Fixed. I should be called `handle_drpc_request_attestation`